mappings/net/minecraft/client/ToastManager.mapping

```diff
@@ -0,0 +1,20 @@
+CLASS net/minecraft/class_594 net/minecraft/client/ToastManager
```

Comment:
why is it called a toast btw any idea?

Reply:
yarn and bootstrap studio call it toast :tm: